src/components/Thanks/ThanksPageDonationOnly.vue

```diff
@@ -0,0 +1,387 @@
+									<span>Share on LinkedIn</span>
+								</a>
+							</div>
+						</div>
+						<div class="large-2"></div>
+					</div>
+					<div v-if="!isGuest" class="continue-link">
+						<router-link
+							to="/portfolio"
+							v-kv-track-event="['Thanks','click-portfolio-cta','No, continue to my portfolio']"
+						>
+							No, continue to my portfolio
+						</router-link>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import { mdiLink } from '@mdi/js';
+import socialSharingMixin from '@/plugins/social-sharing-mixin';
+import KvIcon from '@/components/Kv/KvIcon';
+import { getFullUrl } from '@/util/urlUtils';
+import GenericPromoBanner from '@/components/WwwFrame/PromotionalBanner/Banners/GenericPromoBanner';
+import { gql } from '@apollo/client';
+import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
+
+const userQuery = gql`query userQuery {
+	my {
+		id
+		userAccount {
+			id
+			firstName
+			inviterName
+			public
+		}
+	}
+}`;
+
+export default {
+	name: 'ThanksPageDonationOnly',
+	inject: ['apollo'],
+	components: {
+		KvIcon,
+		KvMaterialIcon,
+		GenericPromoBanner,
+	},
+	props: {
+		shareAskCopyVersion: {
+			type: String,
+			default: 'a'
+		},
+		monthlyDonationAmount: {
+			type: String,
+			default: ''
+		}
+	},
+	mixins: [socialSharingMixin],
+	data() {
+		return {
+			name: '',
+			mdiLink,
+			copyStatus: {
+				class: '',
+				disabled: false,
+				text: 'Copy Link'
+			},
+			youtubeId: 'Mpp2ZH7os4Q',
+			isGuest: false
+		};
+	},
+	computed: {
+		promoBannerContent() {
+			return {
+				link: 'https://www.kiva.org/settings/subscriptions',
+				// eslint-disable-next-line max-len
+				richText: `<p>Thanks! Every month you’ll get an email confirming your $${this.monthlyDonationAmount} donation.View your <u>subscription settings</u> to review, make changes, or cancel</p>`,
+				iconKey: ''
+			};
+		},
+		shareLink() {
+			const base = `https://${this.$appConfig.host}`;
+			const args = {
+			};
+
+			return getFullUrl(base, args);
+		},
+		headerMsg() {
+			if (this.isGuest) return 'Thank you for supporting our mission';
+			return `${this.name}, thank you for supporting our mission`;
+		},
+		utmContent() {
+			if (this.isGuest) return 'guest';
+			if (this.lender?.public && this.lender?.inviterName) return this.lender?.inviterName;
+			return 'anonymous';
+		},
+		utmCampaign() {
+			if (this.shareAskCopyVersion === 'b') {
+				return 'social_share_checkout_variant';
+			}
+			return 'social_share_checkout_control';
+		},
+	},
+	methods: {
+		gatherCurrentUserData() {
+			this.apollo.query({
+				query: userQuery,
+			}).then(({ data }) => {
+				this.isGuest = !data?.my?.userAccount;
+				this.name = data?.my?.userAccount?.public ? data?.my?.userAccount?.firstName : '';
+			});
+		}
+	},
+	created() {
+		this.gatherCurrentUserData();
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+@import "foundation";
+
+$color-facebook: #3b5998;
```

Comment:
We have social colors defined in styleguide, please use those.
https://github.com/kiva/kv-ui-elements/blob/main/%40kiva/kv-tokens/primitives.json#L60

Reply:
If they do not match, please work with Design to match them up.

Reply:
@christian14b thanks for checking on the vars we have defined in tokens!